torch/onnx/symbolic_opset9.py

```diff
@@ -261,8 +261,19 @@ def _slice(g, input, axes, starts, ends):
     return g.op("Slice", input, axes_i=axes, starts_i=starts, ends_i=ends)
 
 
+def _maybe_cast_reduce_op_input(g, self):
+    dtype = self.type().scalarType()
+    # This check only covers traced modules where dtype is present
+    if dtype is not None:
+        # pytorch reduce-ops cast all other input types to int64
```

Comment:
should it be *all other integral types? float types are not included right?

Reply:
Thanks for catching this. Correct, I fixed it.